lib/taste_tester/tunnel.rb

```diff
@@ -49,9 +49,10 @@ def cmd
       else
         pid = '\\$\\$'
       end
+      @max_ping = @delta_secs / 10
       cmds = "ps -p #{pid} -o pgid | grep -v PGID" +
              " > #{TasteTester::Config.timestamp_file} &&" +
-             " touch -t #{TasteTester::Config.testing_end_time}" +
+             " touch --date '#{TasteTester::Config.testing_end_time}'" +
```

Comment:
I used `" touch -t #{TasteTester::Config.testing_end_time.strftime('%y%m%d%H%M.%S')} " +` instead, as that is how the timestamp  is constructed in `host.rb`